### ISSUE TITLE
The file is not loaded into PRCPS if there are no entries in the MR memory.

### ISSUE THREAD

**Issue by osiemk** [G1LRO/chirp2cxf#2]
A small error.
If the MR memory is empty, the file is incorrectly generated.
Cannot be opened by PRCPS.
One position in the MR memory is enough and everything is OK.
Thank you for a great job.
Regards
SQ9KCU

**Comment by G1LRO** [comment]
So are you saying the donor cxf file needs at least one memory defined? If so then that makes sense as the program looks for the tag at the start of existing memories to replace with the chirp generated entries. Probably not going to fix this but will add a note to the readme. Thanks for letting me know.

**Comment by osiemk** [comment]
Exactly. After resetting the VFO, the MR memory is empty and after adding the chirp2cxf channels, the cxf file was corrupted. Channels were added on the </DevicePara> line. If the cxf configuration does not have MR memory, then <Channels_MR /> appears in the file.
Example:
  <Contacts>
....
  </Contacts>
   <Channels_MR />
   <Channels_VFO>
....
  </Channels_VFO>
Regards
SQ9KCU

**Comment by osiemk** [comment]
Exactly. After resetting the VFO, the MR memory is empty and after adding the chirp2cxf channels, the cxf file was corrupted. Channels were added on the </DevicePara> line. If the cxf configuration does not have MR memory, then <Channels_MR /> appears in the file.
Example:

  <Contacts>
....
  </Contacts>
   <Channels_MR />
   <Channels_VFO>
....
  </Channels_VFO>

Regards
SQ9KCU

**Comment by osiemk** [comment]
Exactly. After resetting the VFO, the MR memory is empty and after adding the chirp2cxf channels, the cxf file was corrupted. Channels were added on the </DevicePara> line. If the cxf configuration does not have MR memory, then <Channels_MR /> appears in the file.
Example:

1 <Contacts>
...
2 </Contacts>
3 <Channels_MR /> ----------------->this line
4 <Channels_VFO>
....
5 </Channels_VFO>

Regards
SQ9KCU
